man/systemd.netdev.xml

```diff
@@ -529,7 +529,7 @@
       <varlistentry>
         <term><varname>Id=</varname></term>
         <listitem>
-          <para>The VXLAN ID to use.</para>
+          <para>The VXLAN ID to use. Takes a number in the range [1-16777215].</para>
```

Comment:
You really don't need braces here. Also use either an n-dash or dots.

Reply:
I like the `[1-nnn]` format. It is mathematically clear.

I cannot remember but have we discussed that in somewhere?

Reply:
Yes much better but updated now according to @keszybz .

Reply:
OK. We have used all `[]`, `..`, `-` in many places. Let's cleanup in another PR.

Reply:
I know of the following syntaxes:
- [a, b] or [a, b), or (a, b] — standard math notation for left-right-inclusive, left-inclusive, right-inclusive ranges
- a – b (note that it's an n-dash in there)
- a .. b (used e.g. by bash)

In standard math I'd interpret `[a-b]` as the floor of the difference of a and b.
But I agree that we can have the discussion in a separate PR.